Packs/GroupIB_ThreatIntelligenceAttribution/IncidentFields/incidentfield-GIB_Card_Number.json

```diff
@@ -1,29 +1,60 @@
 {
-    "associatedToAll": false,
-    "associatedTypes": [
-        "GIB Compromised Card"
-    ],
-    "caseInsensitive": true,
-    "cliName": "gibcardnumber",
-    "closeForm": false,
-    "content": true,
-    "editForm": true,
-    "group": 0,
-    "hidden": false,
-    "id": "incident_gibcardnumber",
-    "isReadOnly": false,
-    "locked": false,
-    "name": "GIB Card Number",
-    "neverSetAsRequired": false,
-    "ownerOnly": false,
-    "required": false,
-    "sla": 0,
-    "system": false,
-    "threshold": 72,
-    "type": "shortText",
-    "unmapped": false,
-    "unsearchable": false,
-    "useAsKpi": false,
-    "version": -1,
-    "fromVersion": "6.0.0"
+	"aliasTo": "",
+	"aliases": null,
+	"associatedToAll": false,
+	"associatedTypes": [
+		"GIB Compromised Card",
+		"GIB Compromised Card Group"
+	],
+	"autoCompleteTags": null,
+	"breachScript": "",
+	"cacheVersn": 0,
+	"caseInsensitive": true,
+	"cliName": "gibcardnumber",
+	"closeForm": false,
+	"columns": null,
+	"content": true,
+	"defaultRows": null,
+	"definitionId": "",
+	"description": "",
+	"editForm": true,
+	"fieldCalcScript": "",
+	"fromServerVersion": "6.0.0",
+	"group": 0,
+	"hidden": false,
+	"id": "incident_gibcardnumber",
+	"isReadOnly": false,
+	"itemVersion": "1.4.2",
+	"locked": false,
+	"mergeStrategy": "",
+	"name": "GIB Card Number",
+	"neverSetAsRequired": false,
+	"openEnded": false,
+	"orgType": "shortText",
+	"ownerOnly": false,
+	"packID": "GroupIB_ThreatIntelligenceAttribution",
+	"packName": "Group-IB Threat Intelligence",
+	"packPropagationLabels": [
+		"all"
+	],
+	"placeholder": "",
+	"propagationLabels": [],
+	"required": false,
+	"runScriptAfterUpdate": false,
+	"script": "",
+	"selectValues": null,
+	"sla": 0,
+	"system": false,
+	"systemAssociatedTypes": null,
+	"template": "",
+	"threshold": 72,
+	"toServerVersion": "",
+	"type": "shortText",
```

Comment:
Just to confirm, was it intentional to have this as Short Text type field? If it includes non numerical characters it would make sense

Reply:
Yes, this type is set intentionally and I think it's better to keep it for smoother integration with our API